src/utils/print.ts

```diff
@@ -1,9 +1,10 @@
+/* tslint:disable:no-console */
 import { ENV } from '@app/env';
 
 /**
  * An array of string, each of which is the name of an environment in which this service will log on the console
  */
-const whitelist: string[] = [
+const whitelist = [
```

Comment:
Why is the type removed ?
I suppose it's a consequence of a TSLint rule, but isn't that an _un_-TypeScript thing to do ?